index.html

```diff
@@ -14,29 +14,76 @@
     <title> Awesome Books </title>
   </head>
   <body>
-    <div class="books-container">
+    <section class="nav-bar-container">
+      <div class="nav-bar">
+        <div class="main-logo">
+          <a href="#"><h3>Awesome Books</h3></a>
+        </div>
+        <nav class="menus">
+          <ul>
+            <li>
+              <a href="#" id="book-list-menu"> List </a>
+            </li>
+            <li>
+              <span class="vertical-line"></span>
+            </li>
+            <li>
+              <a href="#" id="add-new-book-menu"> Add new </a>
+            </li>
+            <li>
+              <span class="vertical-line"></span>
+            </li>
+            <li>
+              <a href="#" id="contact-menu"> Contact </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </section>
+
+    <div class="date-container">
+      <p id="date"></p>
+    </div>  
+
+    <section class="books-container">
       <h1 class="first-title"> List Awesome Books </h1>
       <fieldset class="hide" id="fieldset">
         <div class="container-book-list" id="container-book-list">
           <div class="load-books-table"> </div>
         </div>
       </fieldset>
```

Comment:
[REQUIRED] Your footer is at the centre of the page instead of at the lower part of the page. Kindly fix this.
![image](https://user-images.githubusercontent.com/62938735/211908361-3aeee9be-4f4d-44ea-a327-f6ac64fc6799.png)